examples/fluids/navierstokes.c

```diff
@@ -45,11 +45,14 @@
 const char help[] = "Solve Navier-Stokes using PETSc and libCEED\n";
 
 #include "navierstokes.h"
+#include <ceed/sycl.h>
+#include <petscdevice.h>
 
 #include <ceed.h>
 #include <petscdmplex.h>
 #include <petscts.h>
 
+int CeedSetStream_Sycl(Ceed ceed, void* handle) __attribute__((weak));
```

Comment:
I think it makes sense to put this somewhere in the backend itself. I'm not sure where though.

Reply:
If this needs to be publicly viewable, I'd put it in `include/ceed/sycl.h`, like `include/ceed/cuda.h` and `include/ceed/hip.h`

The name isn't quite our convention. Maybe `CeedSetSyclStream`?

Reply:
Ah, I see. Maybe we put that on the list to tweek in a follow up.

James, can you make an issue with all the things we want to follow up on?

Reply:
I had the thought that this might be with intention of having other GPU backends recreate the same functionality, but wasn't sure if the concept translated across hardware. 

That being said, I actually already changed the name over, but we can change it back if/when we make the global interface.

Reply:
> James, can you make an issue with all the things we want to follow up on?

https://github.com/CEED/libCEED/issues/1230